addons/web/static/src/core/utils/html.js

```diff
@@ -286,3 +286,27 @@ export function setElementContent(element, content) {
         element.textContent = content;
     }
 }
+
+export function isMarkup(value) {
+    return value instanceof Markup;
+}
```

Comment:
This is used only here, no need to export it. Probably don't even need to make it a function. The file makes use of this in several place, but repeating `instanceof Markup` is fine.